src/utils/zendesk.ts

```diff
@@ -1,13 +1,42 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
If we have a legitimate need to disable checks for `@typescript-eslint/no-explicit-any` here, rather than finding a better type to use that isn't `any`, then we should be doing it with the minimal possible scope (eg. disable it only for the line/lines that it occurs on, not the whole file)

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/105